apps/gp2-server/src/controllers/user.controller.ts

```diff
@@ -96,6 +96,7 @@ export default class Users implements UserController {
       {
         email: user.email,
         connections: [...(user.connections || []), { code: authUserId }],
+        activatedDate: user.activatedDate ?? new Date().toISOString(),
```

Comment:
whats the scenario where we would have a date already?

Reply:
There is. All goes back to the conversations we're had in RTM about connections. 

I think a user will be allow multiple connection, so could go past this code more that once. 

If that happens, I think we should keep the first activated date.